### ISSUE TITLE
[Improvement] Full date in logs

### ISSUE THREAD

**Issue by vladiulianbogdan** [fpillet/NSLogger#280]
Hi,

My team is using NSLogger also to display logs that were generated by field users. This means that the file might contain multiple days of logging. Thus, it would be really helpful to display also the day / month / year in the log message in UI.

I understand that this might take some width space from the actual message text. If that is a problem, we could have a button like "Show / hide function names" that toggles between a short date format and a long date format that also contains the day, month, year information.

If you think this change could be beneficial, I could make a PR with the change.

Thanks!

**Comment by fpillet** [comment]
Oops sorry, realizing I didn't answer this question.

I've had a discussion about this very feature with @stuffmc. I think the space problem can be solved by inserting markers every time the date changes. Optionally, we could make the marker stick at top while scrolling (or provide context any other way, i.e. in the bottom bar).

What do you think?

**Comment by vladiulianbogdan** [comment]
Yes, I think the marker + the marker stick at the top would work. Something similar to what Calendar application from mac OS is doing when Month view is selected. That would be awesome.
